src/plugins/intel_gpu/src/kernel_selector/cl_kernels/fully_connected_gpu_bf_tiled.cl

```diff
@@ -38,10 +38,17 @@
 #   error "fully_connected_gpu_bf_tiled.cl - TILE_K_OFM must be equal to TILE_K * TILE_OFM and at most 8"
 #endif
 
+#if COMPRESSED_WEIGHTS_INT4
+#   if TILE_K_OFM != TILE_K_OFM_PACKED * 2
+#       error "fully_connected_gpu_bf_tiled.cl - TILE_K_OFM must be divisible by 2 for NF4 compressed case"
```

Comment:
Update error message: "...divisible by 2 for 4-bit compressed case"

Reply:
updated